packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEditPanel.tsx

```diff
@@ -37,16 +38,22 @@ interface InboundLineEditPanel {
   updateDraftLine: (patch: Partial<DraftInboundLine> & { id: string }) => void;
 }
 
+const expiryInputColumn = getExpiryDateInputColumn<DraftInboundLine>();
```

Comment:
Bit of a gotcha here is that whenever the `updateDraftLine` function changes (When an update happens), we need to get new columns, which means if this is not a stable reference to an object, means the `Cell` will be re-mounted, rather than re-rendered which means it will lose all internal state